files/zh-cn/web/css/translate/index.md

```diff
@@ -23,77 +25,87 @@ translate: 50% 105px;
 
 /* Three values */
 translate: 50% 105px 5rem;
+
+/* Global values */
+translate: inherit;
+translate: initial;
+translate: revert;
+translate: revert-layer;
+translate: unset;
 ```
 
 ### 值
 
-- 单个长度/百分比值
-  - : 一个长度值或百分比，表示二维平移，与声明了 X 轴和 Y 轴的平移一样（此时省略的第二个值默认为 0）。等同于在 `translate()` 函数（2D 平移）中指定单个值。
-- 两个长度/百分比值
-  - : 两个长度值或百分比表示在二维上分别按照指定 X 轴和 Y 轴的值进行的平移。等同于在 `translate()` 函数（2D 平移）中函数指定两个值。
-- 三个长度/百分比值
-  - : 三个长度值或百分比，表示分别指定 X 轴、Y 轴、Z 轴的值进行三维平移。等同于`translate3d()` 函数（3D 平移）。
+- 单个 {{cssxref("&lt;length-percentage&gt;")}} 值
+  - : 一个 {{cssxref("&lt;length&gt;")}} 或 {{cssxref("&lt;percentage&gt;")}}，指沿 X 轴平移。等同于在 `translate()` 函数（2D 平移）中指定单个值。
+- 两个 {{cssxref("&lt;length-percentage&gt;")}} 值
+  - : 两个 {{cssxref("&lt;length&gt;")}} 或 {{cssxref("&lt;percentage&gt;")}} 表示在二维上分别按照指定 X 轴和 Y 轴的值进行的平移。等同于在 `translate()` 函数（2D 平移）中函数指定两个值。
+- 三个值
+  - : 两个 {{cssxref("&lt;length-percentage&gt;")}} 和 单个的 {{cssxref("&lt;length&gt;")}} 分别指定 X 轴、Y 轴、Z 轴的值进行三维平移。等同于`translate3d()` 函数（3D 平移）。
 - `none`
   - : 表示不应用平移效果。
 
+## 形式定义
+
+{{CSSInfo}}
+
 ### 形式语法
 
-{{csssyntax}}
+{{CSSSyntax}}
 
 ## 示例
 
-### HTML
+### 悬停移动元素
+
+此示例显示如何使用 `translate` 属性在 X、Y、Z 三个轴上移动元素。
+第一个盒子沿 X 轴移动，第二个沿 X、Y 轴方向，第三个盒子沿着 X、Y、Z 轴，并且由于向父元素添加了 {{cssxref('perspective')}}，看起来是向着观察者移动。
+
+#### HTML
 
 ```html
-<div>
-  <p class="translate">Translation</p>
+<div class="wrapper">
+  <div id="box1">translate X</div>
+  <div id="box2">translate X,Y</div>
+  <div id="box3">translate X,Y,Z</div>
```

Comment:
This will cause text to wrap

Reply:
What do you mean?

Reply:
> What do you mean?

I got it. As a workaround, could you also update the div's `width`. Set it to `9em` would resolve this.

Reply:
I suggest using `min-width: 7em;` instead of fixing width in `.wrapper > div`, in case we are localizing the example. Could you change the en-US content? @fuchunhui

Reply:
Hey @jasonren0403. I believe the fixed width is used to build a square shape. We can modify the CSS ruleset to workaround this. We have used to do the same fix in some CSS pages.

Reply:
> Hey @jasonren0403. I believe the fixed width is used to build a square shape. We can modify the CSS ruleset to workaround this. We have used to do the same fix in some CSS pages.

I'm not sure if square shape is really needed for demonstration here 🤔

Reply:
Yes, the square shape is not actually required, but this is an unnecessary modification to upstream anyway. In addition, it would be more beautiful to keep all those shapes the same size.

Reply:
Okay, let's go on setting `width` and `line-height`